### DISCUSSION TITLE
Tillæg til spotpris, NRGi

### DISCUSSION THREAD

**Discussion by MunchHansen** [rndfm/elspotpris#135] [General]
Er der nogen, der kan forklare hvorfra "Tillæg til spotpris" kommer fra, når man vælger aftaler fra NRGi?

Jeg har ikke kunnet støve tillægget på 0,048 DKK frem nogen steder på NRGis' hjemmeside?

**Comment by rndfm** [comment]
Det er desværre ikke unormalt at elselskaberne ikke oplyser deres tillæg. Lige præcis NRGI ville heller ikke oplyse det hos deres kundeservice. 

Så måden tillægget er blevet fundet hos NRGI er ved at regne elregninger baglæns. 
I princippet kan NRGI have ændret deres tillæg mange gange siden.

**Reply by MunchHansen** [reply]
Hej Christian.

Jeg har en regning her, hvor man ikke kan se andre tillæg end de normale, som jeg også har kunnet finde rundt omkring.

Jeg er ikke ekspert på området, men tænker ikke der ligger noget yderligere skjult?![image](https://user-images.githubusercontent.com/6371955/215279177-960e51f1-4fa5-4306-90b5-59bf52409357.jpeg)

**Comment by rndfm** [comment]
Tillægget er skjult i gennemsnitsprisen på 2,2636.

Hvis du henter dit forbrug for perioden på eloverblik.dk(time opløsning) og sender til mig. Så skal jeg fortælle dig hvad din gennemsnitspris ville være uden tillæg.

**Reply by MunchHansen** [reply]
Hej igen.

Skal du bruge:

Målepunktsdata
Måledata = forbrug
Måleraflæsning
Eller - Prisdata?

Hvor kan jeg sende det til dig? 😃 

Kan man lave en smart udregning af det? Evt kan du måske vise mig et eksempel på udregningen?

**Reply by rndfm** [reply]
Det er måledata/forbrug. Pr time.

Udregningen er nem men lidt besværlig.

Man tager bare forbruget for hver time og ganger med prisen for den time. Totalen dividerer man så med det samlede forbrug for perioden. Så har man gennemsnitsprisen.

**Reply by rndfm** [reply]
Send gerne på christian@critech.dk
